README.md

```diff
@@ -17,11 +17,14 @@ This repo is for Spring Boot Starters of Azure services. It helps Spring Boot de
 
 Below starters are available. You can find them in [Maven Central Repository](https://search.maven.org/).
 
+- [azure-ad-integration-spring-boot-starter](activedirectory/azure-ad-integration-spring-boot-starter/README.md)

```

Comment:
One comment, it will be more helpful if you put a Mavan Central Repo badge for each package.

Reply:
Nice suggestion. Will do in next PR. :)